test/extensions/automated_settings.py

```diff
@@ -6,7 +6,7 @@
 ###
 
 
-def make_simple_cnn_setting(act_cls, bias):
+def make_simple_act_setting(act_cls, bias):
```

Comment:
Is this function still used, or can it be removed?

Reply:
It is in use for the activation settings.